ui/raidboss/data/07-dt/raid/r5s.ts

```diff
@@ -52,6 +52,7 @@ export interface Data extends RaidbossData {
   deepCutTargets: string[];
   storedABSideMech?: 'lightParty' | 'roleGroup';
   discoInfernalCount: number;
+  seenFunkyFloor: boolean;
```

Comment:
Hmm, I think this was removed before being merged in a prior PR.  Does this PR maybe need a rebase?